resources/definitions/anycubic_chiron.def.json

```diff
@@ -0,0 +1,80 @@
+{
+    "version": 2,
+    "name": "Anycubic Chiron",
+    "inherits": "fdmprinter",
+    "metadata":
+    {
+        "visible": true,
+        "author": "Patrick Glatt",
+        "manufacturer": "Anycubic",
+		"category": "Other",
+        "file_formats": "text/x-gcode",
+        "icon": "icon_ultimaker2",
+        "platform": "anycubic_chiron_platform.obj",
+		"platform_texture": "anycubic-chiron.png",
+        "has_materials": true,
+		"preferred_material": "generic_pla",
+        "has_machine_quality": true,
+		"quality_definition": "anycubic_chiron",
+		"preferred_quality_type": "normal",
+        "machine_extruder_trains":
+        {
+            "0": "anycubic_chiron_extruder_0"
+        },
+		"firmware_file": "CHIRON.hex" 
```

Comment:
Your pull request doesn't include this file, so there is no point in defining this.

Reply:
Thx, y in my local cura i had this firmware file. In the fork the firmware folder was missed. 
Should i add the folder including the firmware file or is there a reason why the firmware folder is missed in fork?

Reply:
Firmware files go in a separate repo:
https://github.com/Ultimaker/cura-binary-data/tree/master/cura/resources/firmware

MarlinChiron.hex would probably be a better filename.

Reply:
Thx a lot. I added a pull request on https://github.com/Ultimaker/cura-binary-data/pulls and updated the definition file.
I also saw that somehow my indentation is made with spaces instead of tabs. Maybe it’s because I used a anycubic i3 Mega or similar definition file as template for the chiron definition file. 
Me, I would prefer tabulator. I never use spaces. What do you prefer here?